src/rest/AnalyticsEndpoint.ts

```diff
@@ -36,6 +36,8 @@ export class AnalyticsEndpoint {
   private organization: string;
   public endpointCaller: AnalyticsEndpointCaller;
 
+  private dead: boolean = false;
```

Comment:
You can leave it here but Typescript will understand that the type of the variable is a boolean (or any other type) when you initialize ii with a value.

Reply:
This is resolved as I removed the ability to disable [AnalyticsEndpoint](../tree/master/src/rest/AnalyticsEndpoint.ts) directly.